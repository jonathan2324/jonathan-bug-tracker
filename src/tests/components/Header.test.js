import React from 'react'
import Header from '../../components/HeaderPage'
import ReactShallowRenderer from 'react-test-renderer/shallow'
import { render } from '@testing-library/react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

test('should render header correctly', () => {

    const wrapper = shallow(<Header />)
    expect(toJSON(wrapper)).toMatchSnapshot()
    // const renderer = new ReactShallowRenderer()
    // renderer.render(<Header />)
    // expect(renderer.getRenderOutput()).toMatchSnapshot()
    
})

