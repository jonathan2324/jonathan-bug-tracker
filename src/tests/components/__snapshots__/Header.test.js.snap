// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<div>
  <h1>
    Bug Tracker
  </h1>
  <[object Object]
    to="/"
  >
    Dashboard
  </[object Object]>
  <[object Object]
    to="/create"
  >
    Add Bug
  </[object Object]>
</div>
`;
